### DISCUSSION TITLE
How does pockethost prevent ddos, and how does it distribute traffics?

### DISCUSSION THREAD

**Discussion by txstc55** [pockethost/pockethost#200] [Q&A]
So if I created a database, is it just one instance on one server, or there will be multiple servers communicating with each other? In other words, is my database distributed?

**Comment by benallfree** [comment]
Hi @txstc55, PocketHost provisions one PocketBase process per instance created using the dashboard. Like PocketBase itself, your instance is not distributed to multiple servers. 

DDoS is an interesting idea, but it hasn't really surfaced as a problem yet. Let me know if you have more thoughts though!
